apps/web/src/pages/SearchRequests/SearchPage/components/SearchPools.tsx

```diff
@@ -1,21 +1,25 @@
 import * as React from "react";
 import { useIntl } from "react-intl";
 
-import { Button } from "@gc-digital-talent/ui";
-import { getLocale } from "@gc-digital-talent/i18n";
+import { Button, ExternalLink, Pill } from "@gc-digital-talent/ui";
+import { getLocale, getLocalizedName } from "@gc-digital-talent/i18n";
 
-import { getFullNameHtml } from "~/utils/nameUtils";
 import { getFullPoolAdvertisementTitleHtml } from "~/utils/poolUtils";
 import { SimpleClassification } from "~/types/pool";
-import { Pool } from "~/api/generated";
+import useRoutes from "~/hooks/useRoutes";
+import { PoolAdvertisement } from "@gc-digital-talent/graphql";
+import { insertBetween } from "@gc-digital-talent/helpers";
 
 const testId = (text: React.ReactNode) => (
   <span data-testid="candidateCount">{text}</span>
 );
 
 export interface SearchPoolsProps {
   candidateCount: number;
-  pool: Pick<Pool, "id" | "owner" | "name" | "description" | "classifications">;
+  pool: Pick<
+    PoolAdvertisement,
+    "id" | "owner" | "name" | "classifications" | "team" | "essentialSkills"
```

Comment:
removed description as we don't need it anymore

Reply:
Is the description field deprecated? It doesn't seem to be marked as such in the schema, should it be `@deprecated` ?

Reply:
yeah I think it should be marked as is. Then it has to be cleaned up with all other references like browse/pools page as well. Should we spin off a sep pr for that?